web_src/js/svg.js

```diff
@@ -66,14 +75,25 @@ export const svgs = {
   'octicon-triangle-down': octiconTriangleDown,
   'octicon-x': octiconX,
   'octicon-x-circle-fill': octiconXCircleFill,
+  'octicon-plus': octiconPlus,
+  'octicon-search': octiconSearch,
+  'octicon-filter': octiconFilter,
+  'octicon-archive': octiconArchive,
+  'octicon-star': octiconStar,
+  'gitea-double-chevron-left': giteaDoubleChevronLeft,
+  'gitea-double-chevron-right': giteaDoubleChevronRight,
+  'octicon-chevron-left': octiconChevronLeft,
+  'octicon-organization': octiconOrganization,
 };
 
+// TODO: use a general approach to manage SVG icons
```

Comment:
I see no issue with current approach. It's a bit clumsy but better than the alternatives (wildcard import specific to bundlers has various issues).

Reply:
Let's leave it to future

Reply:
Would still remove the TODO. It's not specific enough and I don't like useless TODOs littered over the codebase.

Reply:
But it's really difficult to maintain, I really dislike copying every name there again and again.

Reply:
Yes, it's unergonomic, but I see no alternative. Webpack can do wildcard import with a [plugin](https://github.com/fred104/webpack-import-glob-loader), but I don't think it's usable to us unless it exposes the filenames.

Reply:
```suggestion
// TODO: use a more general approach to access SVG icons. At the moment, developers must check, pick and fill the names manually, most of the SVG icons in assets couldn't be used directly.
```

Add a more clear (specific) comment.

I can see a possible solution is to make the icons work with Vue reactive system, then they could be loaded by promise. I haven't looked into the details yet.

Reply:
Current solution is ideal for performance as the icons will be embedded as string in the JS. Any kind of async lazy-loading will require incur extra HTTP requests, which I think is not something that is warranted for such small content.

Reply:
Sure, that's why I haven't touched the code, I haven't got a clear picture yet.

And you see, this PR will definitely conflict #23421 , I will resolve the conflicts manually, and remove duplicate entries ...

I will keep in mind about this problem.